optimus/wdl_tasks/Attach10xBarcodes.wdl

```diff
@@ -0,0 +1,31 @@
+
+task Attach10xBarcodes {
+  File r1
+  File i7
+  File u2
```

Comment:
I am liking the comments you put on inputs for other tasks. Could you do that here as well?

Reply:
👍